autoload/go/def.vim

```diff
@@ -126,6 +126,11 @@ function! go#def#jump_to_declaration(out, mode, bin_name) abort
   let out = split(final_out, go#util#LineEnding())[0]
   if go#util#IsWin()
     let parts = split(out, '\(^[a-zA-Z]\)\@<!:')
+  elseif system('uname') =~ 'MINGW' || system('uname') =~ 'CYGWIN'
+    if l:msguri[2:3] is# ':/'
```

Comment:
`l:msguri` doesn't exist. What is this supposed to be? Is it supposed to be `l:out`?

Reply:
Yes, my mistake. It should be `l:out`. Thanks.